web-console/e2e-tests/tutorial-batch.spec.ts

```diff
@@ -195,6 +191,7 @@ async function validateQuery(page: playwright.Page, datasourceName: string) {
     /* page */ 'Talk:Oswald Tilghman',
     /* regionIsoCode */ 'null',
     /* regionName */ 'null',
+    /* time */ '2015-09-12T00:46:58.771Z',
```

Comment:
I updated the e2e test and realized/was faced with this interesting side-effect:
If you specify the time column via the timestampSpec then that column gets excluded from the dimension auto-detection system. If instead you use a transform to construct a `__time` dimension then (obviously) the component dimensions do not get removed from the auto detection. In this case I set `__time` to be `timestamp_parse("time") + 1` and now clicking though all the defaults yields an extra field, the original time column.

Reply:
Does this cause a problem? It's documented behavior (item 3): https://druid.apache.org/docs/latest/ingestion/index.html#inclusions-and-exclusions

Reply:
No, it is not a problem. Just something that took me by surprise. Simply because I did not put 1 + 1 together. Maybe there should be some docs of this inside the web console.